services/workbench-bff/src/api/drift_routes.py

```
@@ -101,9 +101,14 @@ async def scan_for_drift(
     try:
-        result = detector.scan_all_with_persistence(contexts=contexts)
+        result = detector.scan_all_with_persistence(
+            contexts=contexts,
+            require_persistence=True,
+        )
     except Exception as exc:
-        logger.exception("Failed to scan drift with persistence (contexts=%s): %s", contexts, exc)
+        logger.exception(
+            "Failed to scan drift with persistence (contexts=%s): %s", contexts, exc
+        )
         raise HTTPException(status_code=500, detail=str(exc))
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)